dataRetriever/txpool/shardedTxPool_test.go

```diff
@@ -293,11 +295,13 @@ func TestCleanupSelfShardTxCache_NilMempool(t *testing.T) {
 	})
 }
 
-func Test_CleanupSelfShardTxCache(t *testing.T) {
+func Test_Parallel_CleanupSelfShardTxCache(t *testing.T) {
+	_ = logger.SetLogLevel("*:DEBUG")
```

Comment:
Please remove this line. While useful for debugging a specific test, it messes with all the other tests (could hurt during CI).